### ISSUE TITLE
Re-enable installer Repair

### ISSUE THREAD

**Issue by slynch8** [slynch8/10x#942]
Repair has been temporarily removed because it appears the WIX support for this is broken. 

Repair fails if:
1. Original installer exe doesn't exist anymore. This can happen if user deletes installer exe or if temp folder is cleared after updating from within 10x. A possible solution for the updating would be to download the update installer exe to the program files install dir and keep it there. But that doesn't help the case where the user deletes the exe.
2. Even if problem 1 was fixed the Repair then fails repairing NetFx46Redist. I'm not using this anymore, I use .Net 4.7.2 now. However there is no support for NetFx472Redist yet. The docs say it is supported but it doesn't compile (I'm on the latest 3.11.2 toolset). I don't actually think I need any .Net install anymore because I think it should be installed on all modern Windows installs anyway. However, removing NetFx46Redist stops the installer building and I've no idea why.

**Comment by slynch8** [comment]
error from reapir:
```
[7D74:A018][2022-02-03T13:52:48]e000: Error 0x80070002: Failed attempt to download URL: 'http://go.microsoft.com/fwlink/?LinkId=560369' to: 'C:\WINDOWS\Temp\{BE76772E-6865-48DA-B4A8-47C821AE08E8}\NetFx46Redist'
[7D74:A018][2022-02-03T13:52:48]e000: Error 0x80070002: Failed to acquire payload from: 'http://go.microsoft.com/fwlink/?LinkId=560369' to working path: 'C:\WINDOWS\Temp\{BE76772E-6865-48DA-B4A8-47C821AE08E8}\NetFx46Redist'
[7D74:A018][2022-02-03T13:52:48]e313: Failed to acquire payload: NetFx46Redist to working path: C:\WINDOWS\Temp\{BE76772E-6865-48DA-B4A8-47C821AE08E8}\NetFx46Redist, error: 0x80070002.
[7D74:A508][2022-02-03T13:52:48]e000: Error 0x80070002: Failed while caching, aborting execution.
```

**Comment by slynch8** [comment]
It seems that NetFx46Redist is needed because the BA app is C#. Although even that uses 4.7.2 now. But anyway, there doesn't seem to be a way to remove NetFx46Redist so the repair will remain broken. Maybe this will be fixed in a future version of Wix.

**Comment by slynch8** [comment]
fixed (0.32.78).

Found a way to install 4.7.2 which fixes the uninstall problem. Problem 1 still exists though, added a new bug: https://github.com/slynch8/10x/issues/944
